paddle/fluid/framework/block_desc.cc

```diff
@@ -13,11 +13,10 @@ See the License for the specific language governing permissions and
 limitations under the License. */
 
 #include "paddle/fluid/framework/block_desc.h"
+#include <queue>
```

Comment:
This line should be above those "" includes.

Reply:
This is the result of clang-format. When I move line16 above line 15, the clang-format will change to this version.

The reason is that `block_desc.h` is Related header, but `<queue>` is C++ library. The order is: Related header, C library, C++ library, other libraries' .h, your project's .h. https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes